model/version.go

```diff
@@ -240,17 +251,22 @@ func (v *Version) GetTimeSpent(ctx context.Context) (time.Duration, time.Duratio
 	return timeTaken, makespan, nil
 }
 
-func (v *Version) MarkFinished(ctx context.Context, status string, finishTime time.Time) error {
+func (v *Version) MarkFinished(ctx context.Context, status string, finishTime time.Time) (bool, error) {
+	res, err := evergreen.GetEnvironment().DB().Collection(VersionCollection).UpdateOne(ctx, bson.M{
```

Comment:
we can maybe consolodate this with the update above into a 
versionupdatemodified function or something